cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java

```diff
@@ -430,7 +430,8 @@ public Long getResult() {
    * @return constructed table name
    */
   public static String getStateStoreTableName(Id.Namespace namespace) {
-    return String.format("cdap.%s.%s.state.store", namespace.getId(), QueueConstants.QueueType.STREAM.toString());
+    return String.format("cdap.%s.system.%s.state.store",
```

Comment:
In the base, I left it like this, so levelDB, inmemory can still use this.
And then only HBase version uses TableId.

Alternative would be to have a method that turns TableId into a full tableName (such as in hbase 94).